lua/lazyvim/plugins/extras/lang/angular.lua

```diff
@@ -0,0 +1,61 @@
+return {
+  recommended = function()
+    return LazyVim.extras.wants({
+      ft = { "html", "typescript" },
```

Comment:
Can you remove the filetypes here?
Otherwise angular will show up as recommended for any typescript/html file.
Having the root dir detection is enough

Reply:
nice, thanks for the response. I ll update the PR, maybe I ll also remove the `nx.json` ? because nx can also be used to other frameworks.